### PR TITLE
Internationalization integration.

### PR DESCRIPTION
**Pull request by ChrisWeissmann**
# Issue: [VUE-38](https://ticketsystem.schul-cloud.org/browse/VUE-38)

## Description

- Added internationalization with the nuxt-i18n plugin

### CODE REVIEW

**Comment by dominikbruechner** [comment]
> We should also create a UI for the User to change the locale, autodetection based on the browser settings and the possibility to store the locale settings

z.b. with lokalise.co? maybe there are opensource options for that....

File: docs/1-Tutorials/Localization.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+Das aktualisieren kann entweder direkt in den JSON.Objekten unter `./locales/*.json` erfolgen oder über das [lokalise.co Projekt](https://lokalise.co). Wenn Strings über localise aktualisiert wurden, erstellt lokalise einen pull request auf Github der reviewed und approved werden muss.
+
+#### localise Tutorial
+
+1. Einloggen bei [lokalise.co](https://lokalise.co/). (Account Daten momentan bei Dominik)
+2. Project Schulcloud auswählen.
+3. Oben rechts: Key hinzufügen
+4. Key gemäß Namenschema angeben
+5. Die "Base language" ist Deutsch. "Base language value" ist der eigentliche String und der hier in Deutsch eingegeben wird. Platforms sollte standardmäßig auf Web sein.
```
Comment (adrianjost) [inline docs/1-Tutorials/Localization.md]:
```suggestion
5. Die "Base language" ist Deutsch. "Base language value" ist der eigentliche String und soll hier in Deutsch eingegeben werden. Platform sollte standardmäßig auf Web sein.
```

File: docs/1-Tutorials/Localization.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+6. Save
+7. Übersetzung für den Text eingeben. (Google translate ist zu hilfe schon eingebaut)
```
Comment (adrianjost) [inline docs/1-Tutorials/Localization.md]:
```suggestion
7. Übersetzung für den Text eingeben. (Google Übersetzer ist zur Hilfe schon eingebaut)
```

File: src/components/LanguageSwitcher.unit.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+// let store;
+
+// beforeEach(() => {
+// 	store = localStore;
+// });
+
+describe("@components/LanguageSwitcher", () => {
+	it(...isValidComponent(LanguageSwitcher));
+
+	// it("Switch language", () => {
```
Comment (adrianjost) [inline src/components/LanguageSwitcher.unit.js]: After we completed this, we can merge the pull 🎉 🥳

**Pull request merged**